CHANGELOG.md

```diff
@@ -3,7 +3,13 @@
 All notable changes to the COBOL Language Support extension are documented in this file.
 
 ## [0.x.x] - TBD
-
+- Modify dep file creation/update process:[#209](https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/209) Thanks @[zacanbrcom](https://github.com/zacanbrcom)
+- Retrieve settings.json at initialize [#207](https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/207) Thanks @[sergiuilie](https://github.com/sergiuilie)
+- Introduce Code Action to Resolve Copybooks [#198](https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/198) Thanks @[temanbrcom](https://github.com/temanbrcom)
+- Make CobolVisitor constructor-configurable to simplify DI [#205](https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/205) Thanks @[temanbrcom](https://github.com/temanbrcom)
```

Comment:
I think this small refactoring should not appear in the changelog

Reply:
I already planned a small effort to review all the changelog items before the new release, we'll do it in any case also to better check if we miss smth

Reply:
Ok, I agree